onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -758,7 +758,7 @@ struct MLAS_CONV_SYM_PARAMS {
     const int32_t* Bias;
     const float* Scale;
     bool PerChannelScale;
-    uint8_t OutputZeroPoint;
+    int32_t OutputZeroPoint;
```

Comment:
Should ConvDepthwise() parameters of zeropoint change to int32_t also? (at lines around 706)